Library/Formula/cdrtools.rb

```diff
@@ -4,7 +4,10 @@ class Cdrtools < Formula
 
   stable do
     url "https://downloads.sourceforge.net/project/cdrtools/cdrtools-3.01.tar.bz2"
+    mirror "https://www.mirrorservice.org/sites/downloads.sourceforge.net/c/cd/cdrtools/cdrtools-3.01.tar.bz2"
+    mirror "https://fossies.org/linux/misc/cdrtools-3.01.tar.bz2"
     sha256 "ed282eb6276c4154ce6a0b5dee0bdb81940d0cbbfc7d03f769c4735ef5f5860f"
+    patch :p0, :DATA
```

Comment:
`3.02a02` is a preview of the next stable release, `3.02`. The bug in `isoinfo.c` is a write-permission bug and only affects the extraction feature of isoinfo when it is run by non-root users. The other bug, which is due to `clang`, is critical because it makes `isoinfo`  segfault. So we really need both halfs of the patch: the first half to fix the write-permission bug; and the second half to workaround the bug in `clang`.